images/cilium/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# syntax=docker/dockerfile:1.1-experimental
+
+#FROM quay.io/cilium/hubble:v0.6.0 as hubble
+
+FROM --platform=linux/amd64 docker.io/errordeveloper/cilium-builder:019538f3bd3275e8d8369e2c5823efbe0f3a4f06-dev as builder
```

Comment:
what's the process for us to replace `errordeveloper/cilium-builder` with `cilium/cilium-builder`?

Reply:
Still a WIP PR, please ignore this. It's a topic to discuss.